lib/GameConstants.h

```diff
@@ -1241,6 +1241,27 @@ class BattleField : public BaseForID<BattleField, si32>
 	DLL_LINKAGE static BattleField fromString(const std::string & identifier);
 };
 
+class TransportId

```

Comment:
Why this is ID_LIKE and not Identifier?

Reply:
what do you mean?

Reply:
For example - in this file you can find ETerrainId and TerrainId, which done by identifiers.

Reply:
fixed, thanks!